tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -274,15 +284,15 @@ def register_subsystem(self, subsystem: SubSystem):
             They can be used to retrieve the kinematic data from the dataset
             returned by :meth:`~convert`.
         """
-        state_fs = []
+        state_fs: list = []
         for fs_uid in subsystem.final_states:
             state_fs += fs_uid
         invmass_name = self.register_invariant_mass(list(set(state_fs)))
         angle_names = self.register_helicity_angles(subsystem)
 
         return (invmass_name,) + angle_names
 
-    def _convert_ids_to_indices(self, ids: tuple):
+    def _convert_ids_to_indices(self, ids: tuple) -> Sequence:
         """Convert unique ids to event indices.
 
         Uses the :attr:`_fs_id_event_pos_mapping`.
```

Comment:
Interesting, does this render in the API (and does `linkcheck` complain if there's a typo) for 'private' attributes as well?

Reply:
Good point. Since its not used in the docs, it also did not throw any errors on mistakes in the docstring. So I removed the docstring completely